docs/docs/unreachable.md

```diff
@@ -0,0 +1,90 @@
+which is 1 node every 20 mins. With this default, the loss of 2 nodes would result in the first node reporting 5 mins after health checks start to fail that it is unreachable, followed by the 2nd node reporting unreachable 20 mins later. In this scenario, agents and tasks will be reported by Mesos as active for the time between the start of health check failure and the unreachable event as defined here.
+
+The net result is that the amount of time that a node on DC/OS can be lost or unreachable before Marathon is notified is not predictable. It is defined by the number of lost nodes and the order in which they are lost. This was confirmed in a test where a 5 private agent cluster had a task on 1 node. The agent processes on each node were shutdown with the node hosting the task being shutdown last. The amount of time lapsed prior to Marathon receiving a TASK_UNREACHABLE status update was > 1.25 hrs. It is important to understand that these tasks are listed by Mesos as Active until this time and Marathon is completely unaware until notified by Mesos.
+
+**NOTE:**
+There are 2 JIRAs against Mesos to remove rate limits. One marked as a critical bug https://issues.apache.org/jira/browse/MESOS-7721
+the other as a major improvement. https://issues.apache.org/jira/browse/MESOS-5948. It seems as though the default Mesos behavior is as if there is no rate limit, which begs the question why we don't set DC/OS to not have this rate limit?
+
+## Marathon Unreachable Strategy
+
+Marathon has configuration options for working with unreachable tasks by setting the unreachable strategy for a Marathon application as part as the app definition:
+
+```
+unreachableStrategy: {
+"inactiveAfterSeconds": 0,
+"expungeAfterSeconds": 0
+}
+```
+
+In order for this to take effect it is necessary to get a TASK_UNREACHABLE status update from Apache Mesos. The above constraints set the minimum time of managing an unreachable strategy event. The configuration for the app definition sets up to time events. The first is `inactiveAfterSeconds` which is the time in seconds after the TASK_UNREACHABLE event from Mesos that Marathon will create a replacement task. The second configuration is `expungeAfterSeconds` which is the amount of time after the TASK_UNREACHABLE that a task will be removed killed provided 1) it was replaced with the inactiveAfterSecond event and 2) the original task (which was unreachable) becomes reachable again. From the time of `inactiveAfterSeconds` until the "expunge" event, the app will report in Marathon 2 of 1 for the number of tasks running for that app. "expunge" is a little odd, in that it is the time from TASK_UNREACHABLE. If the "inactiveAfterSeconds" trigger event trips, then the expunge will happen in the time configured regardless of if the task becomes reachable or not. In the case of a large delay in the
```

Comment:
s/be removed killed provided/be removed provide/

Reply:
I read this paragraph twice and I don't think I understand it. I think you should revisit it.

Reply:
I think it should be `removed and killed provided 1 and 2`. BTW, does anybody know if `killSelection` applies here? Does it always kill the task that was unreachable?

Reply:
We have inconsistent use of back ticks. :-) Also, inconsistent pluralization. I.e.: inactiveAfterSecond and `inactiveAfterSeconds` :-)